lemming.go

```diff
@@ -26,26 +25,26 @@ import (
 	fp4rt "github.com/openconfig/lemming/p4rt"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/reflection"
+	"k8s.io/klog"
```

Comment:
"k8s.io/klog/v2"? Also should InitFlags(nil) be called? https://github.com/kubernetes/klog#how-to-use-klog However if we do call it I'm wondering if we're running into this problem: https://github.com/kubernetes/klog#coexisting-with-glog since other parts of the code base currently uses glog. So I'm wondering if we should convert to klog, if we choose to do so, all at once in a separate PR?

Reply:
we should switch all our code to use one and i would vastly recommend klog for everything